tests/test_files/calculator/calculator.py

```diff
@@ -10,10 +11,12 @@ def pars_arguments():
     parser = argparse.ArgumentParser()
```

Comment:
This file looks really great, just please make two addition. Add prog and description to the parser like this:
parser.prog = 'some program name'
parser.description = 'some program description'